_includes/partial/user_guide.html

```diff
@@ -1,4 +1,4 @@
 <div>
-  <p>The BCDA Partially Adjudicated Claims User Guide contains information found on this page and additional useful information for REACH ACOs. You can download the User Guide <a href="/assets/data/BCDA_Partially_Adjudicated_Claims_Enhancement_User_Guide_v1.0.0.pdf" target="blank" class="in-text__link">here</a>.
+  <p>The BCDA Partially Adjudicated Claims User Guide contains information found on this page and additional useful information for REACH ACOs. You can download the User Guide <a href="/assets/data/BCDA_Partially_Adjudicated_Claims_Enhancement_User_Guide_v1.2.0.pdf" target="blank" class="in-text__link">here</a>.
```

Comment:
I still get an error when attempting to open the new file; do you get the same thing when you run locally and is that expected behavior when running locally?

Reply:
@laurenkrugen-navapbc I updated the PR description to attach a screen recording of what I am seeing from my end.